### DISCUSSION TITLE
Add topics to repositories for further management

### DISCUSSION THREAD

**Discussion by Tuanm** [untangledfinance/untangled-docs#8] [Ideas]
Thinking we need to add more topics right in our repositories so as to make our further management easier, I swear.

P/s: It looks more professional indeed. 😆

**Comment by Tuanm** [comment]
Topics have been added for all back-end repositories.

**Comment by untangled-finance** [comment]
I have not seen how this is better than slack but we trust you

**Reply by Tuanm** [reply]
It's a way that allows us to view a group of related repositories, like this [back-end ](https://github.com/search?q=topic%3Aback-end+org%3Auntangledfinance+fork%3Atrue&type=repositories) topic.

**Reply by untangled-finance** [reply]
Yes we should become more professional and improve the repos to the point we are comfortable to open source

**Reply by untangled-finance** [reply]
@Tuanm please just go ahead to make this happen - reduce the number of repos to <= 10 monorepos - let me know any questions
